CONTRIBUTING.md

```diff
@@ -1,17 +1,65 @@
-Contributing
-============
+# Contributing
 
-TODO: Fill out your contributing process, or just stick with the default.
-
-Steps
------
+## Steps
 
 Thanks for your interest in sensu-go cookbook! If you'd like to help out please:
 
 1. Fork it
 1. Create your feature branch (`git checkout -b my-new-feature`)
-1. Add tests for the new feature; ensure they pass (`rake`)
+1. Add tests for the new feature; ensure they pass See [TESTING.md](./TESTING.md)
 1. Add a changelog entry under `## [Unreleased]` and the appropriate sub header for the type of change per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 1. Commit your changes (`git commit -am 'Add some feature'`)
 1. Push to the branch (`git push origin my-new-feature`)
 1. Create a new Pull Request
+
+## How to Become a Maintainer
+
+### Types of Maintainers
+
+#### SME (Subject Matter Expert)
+
+##### Pre Conditions
+
+To become an SME maintainer you must meet one of the following conditions:
+
+1. Has made significant contributions to a particular repo
+2. Has specific area knowledge related to the content.
+
+The following is required:
+
+1. Have MFA (Multi Factor Authentication) enabled on your Github account
+
+##### Process
+
+1. Open an [issue](https://github.com/sensu/sensu-go-chef/issues/new) indicating that you would like to become a maintainer and your subject matter expertise.
```

Comment:
I am honestly not sure what the process should be beyond this, since we need someone at sensu inc with admin permissions to invite to the org 🤔

Reply:
Probably need to tag @cwjohnston or @calebhailey here to give them time to think about it a little